### ISSUE TITLE
Open GL error spam the log

### ISSUE THREAD

**Issue by MeeniMc** [kyrptonaught/quickshulker#32]
## The log is full of errors

```
[23:37:34] [Render thread/ERROR]: ########## GL ERROR ##########
[23:37:34] [Render thread/ERROR]: @ Pre render
[23:37:34] [Render thread/ERROR]: 65539: Invalid key -1
```

The message appears immediately after joining a world; it is not correlated with opening boxes or inventories, or moving the mouse. It does however stops when the game is paused (like in the game menu).

Beside the error-log spam (and it's a lot of messages), there are no other ill effects I can tell. 

## Replicate

* MC 1.17.1
* QS 1.2.2-1.17
* Fabric API 0.37.2

* AMD RX580 win64
* Java 16 GraalVM 21.1

No other mods.

**Comment by kyrptonaught** [comment]
I'm gonna assume you don't have the keybinding bound to anything? Setting the quickshulker keybinding should remove the error. Although the log spam isn't pretty and should probably be fixed

**Comment by MeeniMc** [comment]
I turned off the two 'keybinding while ...' options and that stopped the error, even with the keybind still blank, so an effective fix could be to change the config panel so it is not possible to create incorrect option combos.

**Comment by kyrptonaught** [comment]
This was fixed in 1.3.0. Releasing shortly
